main.go

```diff
@@ -84,6 +85,18 @@ func main() {
 			Short: false,
 		},
 	}
+	coverageUrl := os.Getenv(EnvCoverageURL) 
+	if coverageUrl != "" {
+		newfields:= []Field{
+			{
+				Title: "Test coverage URL",
+				Value: os.Getenv(EnvCoverageURL),
+				Short: false,
+			},
+		}
+		fields = append(newfields, fields...)
```

Comment:
It should be the other way around:
`fields = append(fields, newFields...)`

Reply:
Can you help with it?))

Reply:
I already did, no?
what do you need help with?